gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=28.1.34
+version=28.1.35
```

Comment:
Master is already 28.1.35, please rebase master.

Reply:
thanks, fixed.